cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1806,6 +1806,195 @@ search = | savedsearch "Palo Alto Firewall - Network Compromise - DDoS Attack Pr
 action.cyences_notable_event_action.products = Palo Alto
 
 
+# =============================
+# Fortinet Fortigate Firewall
+# =============================
+
+[Fortigate Firewall - Network Compromise - Fortigate DNS Sinkhole]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 5
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = A Fortigate Firewall DNS alert for when outbound traffic goes to 72.5.65.111. \
+\
+Data Collection - Fortigate Networks firewalls traffic data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` (sourcetype="fortigate_traffic" OR sourcetype="fortigate_utm") dest_ip="72.5.65.111" \
+| stats count, values(dvc) as dvc, dc(src_port) as dc_src_port, values(dest_port) as dest_port, values(url) as url, values(app) as app, values(rule) as rule by src_ip \
+| eval cyences_severity = "medium" \
+| `cs_fortigate_dns_sinkhole_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_fortigate_dns_sinkhole_filter
+action.cyences_notable_event_action.contributing_events = `cs_fortigate` (sourcetype="fortigate_traffic" OR sourcetype="fortigate_utm") dest_ip="72.5.65.111"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dvc, rule, app
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_fortigate` (sourcetype="fortigate_traffic" OR sourcetype="fortigate_utm") dest_ip="72.5.65.111" dvc=$row.dvc$
+action.cyences_notable_event_action.attacker_search = | stats sum(count) as count by src_ip
+action.cyences_notable_event_action.attacker_drilldown = `cs_fortigate` (sourcetype="fortigate_traffic" OR sourcetype="fortigate_utm") dest_ip="72.5.65.111" src_ip=$row.src_ip$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = FortiGate
+
+[Fortigate Firewall - Network Compromise - Fortigate High Threats Alert]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 9,39 * * * *
+description = A Fortigate Firewall alerts for high or critical severity threats. \
+\
+Data Collection - Fortigate Networks firewalls traffic data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -32m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` sourcetype="fortigate_utm" severity IN ("high" "critical") \
```

Comment:
Are we sure about this search?

Reply:
I googled for fortigate_utm logs and it is related to threat. UTM (unified threat management) hence similar to palo, this alert generates for high and critical severity threat events.

Reply:
cool.